plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -1502,34 +1457,6 @@ int put_log_events(struct flb_cloudwatch *ctx, struct cw_flush *buf,
         if (c->resp.payload_size > 0) {
             error = flb_aws_error(c->resp.payload, c->resp.payload_size);
             if (error != NULL) {
-                if (strcmp(error, ERR_CODE_INVALID_SEQUENCE_TOKEN) == 0) {
```

Comment:
`flb_aws_error` and `flb_aws_print_error` are basically the same.

Here were are now only still using `flb_aws_error` to check if we have an error so we choose to either print it with `flb_aws_print_error`  or print the full API payload to debug if we can't parse the error.

If you made a small change to add a return value to `flb_aws_print_error`, it could do the check of whether or not here is an error that we can parse. This simplifies the code.

Reply:
This does make the code simpler. I've updated the code to reflect this suggestion.